pkg/api/controller.go

```diff
@@ -4564,6 +4565,11 @@ func (c *Controller) GetObject(w http.ResponseWriter, r *http.Request, repositor
 		w.Header().Set("Content-Length", fmt.Sprint(entry.Size))
 	}
 
+	// time to first byte - include out part of the processing without the actual data transfer
+	requestTTFBHistograms.
```

Comment:
This is a bit unfortunate: we will get GetObject TTFB on a _different_ metric, possibly with different fields.  But I also understand that if we report it on the _same_ metric then summing that metric will be count GetObject TTFB twice - also unfortunate.

Reply:
True. Note that summing the underlaying block adapters ttfb can include other operations that under the hood perform GetObject that uses the same block adapter.